perfmetrics/scripts/show_results.py

```diff
@@ -0,0 +1,38 @@
+import json
+import sys
+
+if __name__ == '__main__':
+  argv = sys.argv
+  if len(argv) != 2:
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 show_results.py <fio output json filepath>')
+
+  # Opening JSON file
+  file = open(argv[1])
+
+  # returns JSON object as
+  # a dictionary
+  data = json.load(file)
+
+  # Iterating through the json
+  # list
+  for d in data['jobs']:
+      if d["read"]["bw"] != 0 :
+        if d["job options"]["rw"] == "read":
+          print("Filesize: "+ d["job options"]["filesize"])
+          #Read
+          print("Read bw: " + str(d["read"]["bw"]/1000.0) + "MiB/s")
```

Comment:
Does fio give output in KiB/sec ? If yes then we should divide by 1024 (MiB = 1024 * KiB) and if no, then we should divide by 1024 * 1024 (MiB = 1024 * 1024 * B)

Reply:
fixed